### PR TITLE
Fixed contributors names

### PR DESCRIPTION
**Pull request by alexander-sidorov**

### CODE REVIEW

File: validate_contributors.sh [hunk 1]
```diff
@@ -32,16 +32,18 @@ assert_names() {
   dir_names=$(ls -d hw/* | sed -e 's/hw\///g')
   github_names=(
     "alexander_sidorov"
-    "andrei_9325"
```
Comment (APOSHAml) [inline validate_contributors.sh:35]: NO

**Review by lerkin011 — APPROVED** [review]

**Review by APOSHAml — CHANGES_REQUESTED** [review]

**Review by SashaYaroshevich — COMMENTED** [review]
good

**Review by denisasipenko — APPROVED** [review]

File: validate_contributors.sh (hunk 1 above)
Comment (alexander-sidorov) [reply validate_contributors.sh:35]: yes

**Review by Yancharski — APPROVED** [review]

**Review by MaksimPtitski — APPROVED** [review]

**Review by maryiasa — APPROVED** [review]

**Review by vadison90125 — APPROVED** [review]

**Review by Tobkirill — COMMENTED** [review]
Review

**Pull request merged**